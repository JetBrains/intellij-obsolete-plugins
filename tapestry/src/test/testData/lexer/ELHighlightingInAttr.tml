<html xmlns:t="http://tapestry.apache.org/schema/tapestry_5_1_0.xsd">
    <head>
	<t:hidden value="the value is ${rrrrr?.tt.r(77, -5.3, [-3,  -2])}!!!">
              <title>Hello World Page</title>
        </t:hidden>    
    </head>
    <body>
        <a href="#" t:type="pagelink" page="startpage">my link</a>
    </body>
</html>
