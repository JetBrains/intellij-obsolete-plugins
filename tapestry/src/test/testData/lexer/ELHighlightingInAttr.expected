XML_START_TAG_START ('<')
XML_TAG_NAME ('html')
TAG_WHITE_SPACE (' ')
XML_NAME ('xmlns:t')
XML_EQ ('=')
XML_ATTRIBUTE_VALUE_START_DELIMITER ('"')
XML_ATTRIBUTE_VALUE_TOKEN ('http://tapestry.apache.org/schema/tapestry_5_1_0.xsd')
XML_ATTRIBUTE_VALUE_END_DELIMITER ('"')
XML_TAG_END ('>')
XML_WHITE_SPACE ('
    ')
XML_START_TAG_START ('<')
XML_TAG_NAME ('head')
XML_TAG_END ('>')
XML_WHITE_SPACE ('
	')
XML_START_TAG_START ('<')
XML_TAG_NAME ('t:hidden')
TAG_WHITE_SPACE (' ')
XML_NAME ('value')
XML_EQ ('=')
XML_ATTRIBUTE_VALUE_START_DELIMITER ('"')
XML_ATTRIBUTE_VALUE_TOKEN ('the value is ')
${ ('${')
TAP5_EL_IDENTIFIER ('rrrrr')
?. ('?.')
TAP5_EL_IDENTIFIER ('tt')
. ('.')
TAP5_EL_IDENTIFIER ('r')
( ('(')
TAP5_EL_INTEGER ('77')
, (',')
WHITE_SPACE (' ')
TAP5_EL_DECIMAL ('-5.3')
, (',')
WHITE_SPACE (' ')
[ ('[')
TAP5_EL_INTEGER ('-3')
, (',')
WHITE_SPACE ('  ')
TAP5_EL_INTEGER ('-2')
] (']')
) (')')
} ('}')
XML_ATTRIBUTE_VALUE_TOKEN ('!!!')
XML_ATTRIBUTE_VALUE_END_DELIMITER ('"')
XML_TAG_END ('>')
XML_WHITE_SPACE ('
              ')
XML_START_TAG_START ('<')
XML_TAG_NAME ('title')
XML_TAG_END ('>')
XML_DATA_CHARACTERS ('Hello')
XML_WHITE_SPACE (' ')
XML_DATA_CHARACTERS ('World')
XML_WHITE_SPACE (' ')
XML_DATA_CHARACTERS ('Page')
XML_END_TAG_START ('</')
XML_TAG_NAME ('title')
XML_TAG_END ('>')
XML_WHITE_SPACE ('
        ')
XML_END_TAG_START ('</')
XML_TAG_NAME ('t:hidden')
XML_TAG_END ('>')
XML_WHITE_SPACE ('    
    ')
XML_END_TAG_START ('</')
XML_TAG_NAME ('head')
XML_TAG_END ('>')
XML_WHITE_SPACE ('
    ')
XML_START_TAG_START ('<')
XML_TAG_NAME ('body')
XML_TAG_END ('>')
XML_WHITE_SPACE ('
        ')
XML_START_TAG_START ('<')
XML_TAG_NAME ('a')
TAG_WHITE_SPACE (' ')
XML_NAME ('href')
XML_EQ ('=')
XML_ATTRIBUTE_VALUE_START_DELIMITER ('"')
XML_ATTRIBUTE_VALUE_TOKEN ('#')
XML_ATTRIBUTE_VALUE_END_DELIMITER ('"')
TAG_WHITE_SPACE (' ')
XML_NAME ('t:type')
XML_EQ ('=')
XML_ATTRIBUTE_VALUE_START_DELIMITER ('"')
XML_ATTRIBUTE_VALUE_TOKEN ('pagelink')
XML_ATTRIBUTE_VALUE_END_DELIMITER ('"')
TAG_WHITE_SPACE (' ')
XML_NAME ('page')
XML_EQ ('=')
XML_ATTRIBUTE_VALUE_START_DELIMITER ('"')
XML_ATTRIBUTE_VALUE_TOKEN ('startpage')
XML_ATTRIBUTE_VALUE_END_DELIMITER ('"')
XML_TAG_END ('>')
XML_DATA_CHARACTERS ('my')
XML_WHITE_SPACE (' ')
XML_DATA_CHARACTERS ('link')
XML_END_TAG_START ('</')
XML_TAG_NAME ('a')
XML_TAG_END ('>')
XML_WHITE_SPACE ('
    ')
XML_END_TAG_START ('</')
XML_TAG_NAME ('body')
XML_TAG_END ('>')
XML_WHITE_SPACE ('
')
XML_END_TAG_START ('</')
XML_TAG_NAME ('html')
XML_TAG_END ('>')
XML_WHITE_SPACE ('
')
